capstone/src/main/java/com/google/univiz/api/SearchResource.java

```diff
@@ -1,7 +1,22 @@
 package com.google.univiz.api;
 
+import java.util.List;
+
 /**
  * The SearchResoure interface is responsible for all data management/processing specific for the
  * search feature of the UniViz webapp.
  */
-public interface SearchResource {}
+public interface SearchResource {
```

Comment:
Wasn't this just reviews in #57? Mind updating the base of this PR to reflect that?